test/kill.js

```diff
@@ -71,18 +93,9 @@ if (process.platform !== 'win32') {
 	test('`forceKillAfterDelay` should kill after a timeout with the killSignal string', testForceKill, 50, 'SIGINT', {killSignal: 'SIGINT'});
 	test('`forceKillAfterDelay` should kill after a timeout with the killSignal number', testForceKill, 50, constants.signals.SIGINT, {killSignal: constants.signals.SIGINT});
 
-	const testInvalidForceKill = async (t, forceKillAfterDelay) => {
-		t.throws(() => {
-			execa('empty.js', {forceKillAfterDelay});
-		}, {instanceOf: TypeError, message: /non-negative integer/});
-	};
-
-	test('`forceKillAfterDelay` should not be NaN', testInvalidForceKill, Number.NaN);
-	test('`forceKillAfterDelay` should not be negative', testInvalidForceKill, -1);
```

Comment:
Those tests can be run on Windows, so they're moved outside of the "no Windows" condition.